app/src/main/java/org/p2p/wallet/striga/signup/ui/StrigaSignUpSecondStepPresenter.kt

```diff
@@ -0,0 +1,58 @@
+package org.p2p.wallet.striga.signup.ui
+
+import kotlinx.coroutines.launch
+import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
+import org.p2p.wallet.striga.signup.StrigaSignUpSecondStepContract
+import org.p2p.wallet.striga.signup.interactor.StrigaSignupInteractor
+import org.p2p.wallet.striga.signup.repository.model.StrigaSignupDataType
+
+class StrigaSignUpSecondStepPresenter(
+    dispatchers: CoroutineDispatchers,
+    private val interactor: StrigaSignupInteractor,
+) :
+    BasePresenter<StrigaSignUpSecondStepContract.View>(dispatchers.ui),
+    StrigaSignUpSecondStepContract.Presenter {
+
+    override fun firstAttach() {
+        super.firstAttach()
+        launch {
+            initialLoadSignupData()
```

Comment:
They won't be empty. 
1. UiKitEditText saves and restores its state with `savedState` when view is recreated.
2. All the data is saved to db when user leaves the screen, because onStop is being called. If system decides to destroy previous fragment in backstack, presenter will be recreated and data will be loaded again in this method.